internal/pkg/cli/svc_logs.go

```diff
@@ -179,6 +195,14 @@ func (o *svcLogsOpts) Execute() error {
 	if o.limit != 0 {
 		limit = aws.Int64(int64(o.limit))
 	}
+	if o.previous {
+		taskID, err := o.latestStoppedTaskID()
+		if taskID != "" {
+			o.taskIDs = []string{taskID}
+		} else {
+			return err
+		}
```

Comment:
I feel we should log here.
```suggestion
		taskID, err := o.latestStoppedTaskID()
		if err != nil {
			return err
		}
		if taskID == "" {
			log warning
			return nil # return early because we don't need to print any logs.
		}
		o.taskIDs = []string{taskID}
```

Reply:
I have done something similar in the latest commit that you have mentioned here, let me know if that works? Else happy to change as you've mentioned.